workshops/views.py

```diff
@@ -183,8 +183,15 @@ def admin_dashboard(request):
     ).active().prefetch_related('tags')
 
     uninvoiced_events = Event.objects.active().uninvoiced_events()
-    unpublished_events = Event.objects.active().unpublished_events() \
-                                      .select_related('host')
+    unpublished_events = Event.objects \
+        .active().unpublished_events().select_related('host').annotate(
+            num_instructors=Count(
+                Case(
+                    When(task__role__name='instructor', then=Value(1)),
+                    output_field=IntegerField()
+                )
+            ),
+        )
```

Comment:
This is not a problem since we'll filter out cancelled events (#1064). Just add a comment explaining that this annotation produces wrong number for cancelled events and that it's ok, because `unpublished_events()` filters out these events.